quarkus/deployment/src/main/java/org/keycloak/quarkus/deployment/KeycloakProcessor.java

```diff
@@ -102,7 +102,7 @@ void configureHibernate(KeycloakRecorder recorder, HibernateOrmConfig config, Li
      * 
      * @param recorder
      */
-    @Record(ExecutionTime.STATIC_INIT)
+    @Record(ExecutionTime.RUNTIME_INIT)
```

Comment:
Is this change intentional? Wouldn't it have impact on the startup time?

Reply:
Yes, it is ..

After analyzing the flame graph   for a startup I noticed stacks related to loading factory classes during static init. Where changing to runtime (main method) it does not show up. O noticed a slight improvement, thus the change.

Also, I think that initialization makes more sense in the main method after all configuration is processed during static init.

Reply:
Ok, Thanks!